packages/functions/src/donor/CompleteAppointmentHandler.test.ts

```diff
@@ -103,6 +103,40 @@ test("Valid request complete appointment", async () => {
   expect(appointment[0].creatorUserId).toEqual("creatorUserId");
 });
 
+test("Valid request complete appointment with true parameter", async () => {
```

Comment:
2 Small comments:
1. The name of the test is not explanatory enough. What does the true or false mean? Consider something like "With No Show".
2. The two tests you added are identical, except for the status. For tests like this you can use `test.each`. See an example here:
https://github.com/startach/ZM-BloodComponents/blob/fa2be91d73aa46eac0e2a4db6976aafd464ef44b/packages/functions/src/coordinator/DeleteAppointmentsHandler.test.ts#L101-L134